terraform/persistent/resources/main.tf

```diff
@@ -0,0 +1,105 @@
+# SPDX-FileCopyrightText: 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+
+provider "azurerm" {
+  # https://github.com/hashicorp/terraform-provider-azurerm/issues/24804
+  skip_provider_registration = true
+  features {}
+}
+
+terraform {
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+    }
+    secret = {
+      source = "numtide/secret"
+    }
+  }
+}
+
+################################################################################
+
+terraform {
```

Comment:
Duplicate `terraform` blocks! should be combined into a single block to avoid confusion.

Reply:
Fixed in https://github.com/tiiuae/ghaf-infra/commit/eb4eb5ae550bcd4ab71ac97ea1733ba79bb3490f